src/Waives.Extensions.DocumentChannels.Filesystem/FileSystem.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.IO;
+using System.Linq;
+using System.Threading;
+using Waives.Reactive;
+
+namespace Waives.Extensions.DocumentChannels.Filesystem
```

Comment:
✋ We should change this to `Waives.Extensions.DocumentSources.FileSystem` now I think.

Reply:
Will be completed in a separate PR.